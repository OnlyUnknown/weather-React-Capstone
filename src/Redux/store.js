import { configureStore } from '@reduxjs/toolkit';
import mainReducer from './Slices/mainpageSlice';

const store = configureStore({
  reducer: {
    main: mainReducer,
  },
});

export default store;
