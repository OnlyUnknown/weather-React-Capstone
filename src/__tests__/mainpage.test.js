import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import MainPage from '../components/mainpage';
import store from '../redux/store';

describe('Component render testing', () => {
  it('Line page renders correctly', () => {
    const { asFragment } = render(
      <Provider store={store}>
        <MainPage />
      </Provider>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
