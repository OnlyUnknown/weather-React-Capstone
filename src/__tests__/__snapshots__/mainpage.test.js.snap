// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component render testing MainPage page renders correctly 1`] = `
<DocumentFragment>
  <div
    class="background-Nav"
  >
    <ul
      class="Nav-icon"
    >
      <li>
        <img
          alt="HomePage"
          class="sizeI"
          src="https://cdn-icons-png.flaticon.com/512/7133/7133312.png"
        />
      </li>
      <li
        class="center"
      >
        Top 50 cities Daily Forecasts
      </li>
      <li
        class="Left"
      >
        <img
          alt="Mic"
          class="sizeI"
          src="https://cdn-icons-png.flaticon.com/512/1082/1082810.png"
        />
      </li>
      <li
        class="right"
      >
        <img
          alt="Setting"
          class="sizeI"
          src="https://cdn-icons-png.flaticon.com/512/126/126472.png"
        />
      </li>
    </ul>
  </div>
  <div>
    Loading
  </div>
</DocumentFragment>
`;
